www/docs/en/dev/guide/platforms/android/lifecycle.md

```diff
@@ -102,12 +178,16 @@ The possible plugin statuses in the `pendingResult` field include the following
 "Error
 ```
 
->**NOTE:** It is up to the plugin to decide what is contained in the `result` field and the meaning of the `pluginStatus` that is returned. Reference the API of the plugin you are using to see what you should expect those fields to contain and how to use their values
+Please note that it is up to the plugin to decide what is contained in the

```

Comment:
Does the camera plugin have details on what these results mean and how one should respond?